README.md

```diff
@@ -13,30 +13,60 @@ A script to automatically back up all databases running under docker on a host
 
 ## Installation
 
-This container requires access to the docker socket. This can be done either by mounting `/var/lib/docker.sock`, or using a HTTP proxy to provide it through `$DOCKER_HOST`.
+This container requires access to the docker socket. This should always be done, for security reasons, using a [HTTP proxy](https://github.com/Tecnativa/docker-socket-proxy) which this container can then access through the `DOCKER_HOST` environment variable. 
```

Comment:
There's a lot of wording change here, which doesn't seem to add much. I'm ok with talking about the socket before using a proxy (even if it is worse).

I personally always use `$` as a prefix for environment variables, although I agree that might not be clear. I'd be ok with a simpler patch which just rewords around that instead.